### PR TITLE
Add new changes

### PR DESCRIPTION
**Pull request by senyamac**

### CODE REVIEW

File: src/main/java/com/example/springapi/controller/MyRestController.java [hunk 1]
```diff
@@ -1,93 +1,100 @@
...
   }
 
   @RequestMapping("/jokebase")
-  public ResponseEntity<List<Joke>> getAllJokes() {
-    List<Joke> jokes;
-    jokes = new ArrayList<>(jokeService.findAll());
-
-    if (jokes.isEmpty()) {
+  public ResponseEntity<List<JokeEntity>> getAllJokes() {
+    List<JokeEntity> jokeEntities;
```
Comment (Macost) [inline src/main/java/com/example/springapi/controller/MyRestController.java]: please initialize variable the same line where it's declared if possible
check other lines for this requirement

File: src/main/java/com/example/springapi/controller/MyRestController.java [hunk 2]
```diff
@@ -1,93 +1,100 @@
...
+    jokeEntity = jokeService.findJokeById(id);
+    if (jokeEntity == null) {
+      return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+    } else {
+      return new ResponseEntity<>(jokeEntity, HttpStatus.OK);
+    }
+  }
+
+  @RequestMapping("/jokebase/length/{subpath}")
+  public ResponseEntity<String>  getLengthJokeById(@PathVariable("subpath") String subpath) {
```
Comment (Macost) [inline src/main/java/com/example/springapi/controller/MyRestController.java]: I would say Joke have length, and not the opposite
either you can write JokeLength or LengthOfJoke
I prefer 1st option

File: src/main/java/com/example/springapi/controller/MyRestController.java (hunk 1 above)
Comment (senyamac) [reply src/main/java/com/example/springapi/controller/MyRestController.java]: fixed

File: src/main/java/com/example/springapi/controller/MyRestController.java (hunk 2 above)
Comment (senyamac) [reply src/main/java/com/example/springapi/controller/MyRestController.java]: done

**Pull request merged**